tephra-hbase-compat-1.1-base/src/main/java/org/apache/tephra/hbase/coprocessor/TransactionProcessor.java

```diff
@@ -140,29 +141,25 @@ public void start(CoprocessorEnvironment e) throws IOException {
         ttlByFamily.put(columnDesc.getName(), ttl);
       }
 
-      this.allowEmptyValues = env.getConfiguration().getBoolean(TxConstants.ALLOW_EMPTY_VALUES_KEY,
-                                                                TxConstants.ALLOW_EMPTY_VALUES_DEFAULT);
+      String allowEmptyValuesFromTableDesc = tableDesc.getValue(TxConstants.ALLOW_EMPTY_VALUES_KEY);
+      this.allowEmptyValues = (allowEmptyValuesFromTableDesc != null) ? Boolean.valueOf(allowEmptyValuesFromTableDesc) :
+        getConfiguration(env).getBoolean(TxConstants.ALLOW_EMPTY_VALUES_KEY, TxConstants.ALLOW_EMPTY_VALUES_DEFAULT);
```

Comment:
Looks like `allowEmptyValues` was not a table level property before. Any reason why we made it into a table level property now?

> If we do it lazily, then we might get different results for a Get operation based on whether we were able to read the Configuration successfully or not.

If not then it can be initialized incorrectly during startup, and will always be wrong, right? I think it should always be read lazily.